03_letters/anishaNair/main.go

```diff
@@ -0,0 +1,39 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+	"sort"
+)
+
+var out io.Writer = os.Stdout
+
+func main() {
+	sortAndPrint(countLetters("aba"))
+}
+
+func countLetters(s string) map[rune]int {
+	elementMap := make(map[rune]int)
```

Comment:
Cleaner initialisation code: `elementMap := map[rune]int{}`

Reply:
Fixed it